dash.el

```diff
@@ -1554,6 +1554,69 @@ and when that result is non-nil, through the next form, etc."
                    (--> ,result ,form))
                  ,@more))))
 
+(defmacro ---thread-each-and-collect (op x collector &optional forms)
+  "Private: Used by -< and -<<.
+
+Returned expr: (COLLECTOR (OP X form1) (OP X form2) ...).
+If no forms were passed then returned expr would be: (COLLECTOR X).
+X will be evaluated only once.
+"
+  (let ((value (make-symbol "value")))
+    `(let ((,value ,x))
+       (,collector ,@(--map (list op value it)
```

Comment:
shouldn't `value` be unquoted?

Reply:
The whole sexp is unquoted: `,@(--map (list op value...`.

Reply:
of course :) :+1: